lib/views/after_auth_screens/profile/profile_page.dart

```diff
@@ -204,20 +243,7 @@ class ProfilePage extends StatelessWidget {
                             const Tab(text: 'Tasks'),
                           ],
                           views: [
-                            ColoredBox(
-                              color: Theme.of(context).colorScheme.background,
-                              child: GridView.count(
-                                mainAxisSpacing: 5,
-                                crossAxisCount: 3,
-                                children: [
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                  Image.asset('assets/images/pfp2.jpeg'),
-                                ],
-                              ),
-                            ),
+                            IndividualPostView(post: samplePost),
```

Comment:
This is still not covered. Connect with @Ayush0Chaudhary or @Dante291, anyone will help you out.